build/ci.go

```diff
@@ -287,7 +288,31 @@ func doTest(cmdline []string) {
 	if *vet {
 		build.MustRun(goTool("vet", packages...))
 	}
+	if *misspell {
+		// Ensure the spellchecker is available
+		build.MustRun(goTool("get", "github.com/client9/misspell/cmd/misspell"))
 
+		// Windows (AppVeyor) chokes on long argument lists, check packages individualy
```

Comment:
Why should I remove the workaround? It doesn't make anything slower, and at least it would still work if someone runs it locally on Windows?